test/js/bun/util/ms.test.ts

```diff
@@ -0,0 +1,436 @@
+import { describe, expect, test } from "bun:test";
```

Comment:
Please add vercel's tests too. You actually do not pass their test suite completely.

<img width="554" height="146" alt="image" src="https://github.com/user-attachments/assets/55ec3c9f-3646-47f2-8bc7-39602ef1788a" />

I've opened a PR targeting your branch here which adds said tests: https://github.com/oven-sh/bun/pull/23704